melview.py

```diff
@@ -345,6 +356,11 @@ def get_mode(self):
 
         return 'Auto'
 
+    def get_zone(self, zoneid):
```

Comment:
If the below is a dictionary, this can just return the item if present, else None.

Reply:
done!